packages/client/hmi-client/docker/Dockerfile

```diff
@@ -1,14 +1,29 @@
+## Stage 1 : build with gradle builder image with native capabilities
```

Comment:
Comment is mentionning `gradle` but it's using node as builder

Reply:
oops, fixed